pytopo/rf/alazar/acquisition_controllers.py

```diff
@@ -243,16 +259,22 @@ def handle_buffer(self, data, buffer_number=None):
         
         if buffer_number is None or self.average_buffers():
             self.data[self._cur_block_idx*n : (self._cur_block_idx+1)*n] += data
-            self.handling_times[0] = (time.perf_counter() - t0) * 1e3
         else:
             self.data[buffer_number*n : (buffer_number+1)*n] = data
-            self.handling_times[buffer_number] = (time.perf_counter() - t0) * 1e3
+            
+
+        if buffer_number is None:
+            bn = 0
+        else:
+            bn = buffer_number
```

Comment:
`bn` is not used, right? will the code stop doing something important if `buffer_number` is `None`? :)

actually, if i remember correctly, alazar driver always passes an int for `buffer_number`, so it can basically never be `None` (unless the driver code changes...).

Reply:
oh-oh -- apparently i forgot to change the line below (the index of ``handling_times`` cannot be ``None``).
There used to be a version of the alazar driver where the argument could be None -- but apparently that's not the case anymore (and 0 is passed for the first buffer), otherwise this code would have crashed on me :)

If the driver never passes ``None``, then this piece of code can go, otherwise the line below needs to be fixed.